robot/Cumulus/resources/RecurringDonationsPageObject.py

```diff
@@ -37,9 +37,17 @@ def select_value_from_rd2_modal_dropdown(self, dropdown, value):
     @capture_screenshot_on_error
     def populate_rd2_modal_form(self, **kwargs):
         """Populates the RD2 modal form fields with the respective fields and values"""
+        ns=self.npsp.get_npsp_namespace_prefix()
         for key, value in kwargs.items():
+            locator = npsp_lex_locators["erd"]["modal_input_field"].format(key)
+            # Recurring Donation Name field only appears on a regression org hence this check
+            if key == "Recurring Donation Name" and ns=="npsp__":
```

Comment:
Just out of curiosity why this field is only on Regression org? Would love to learn during our sync up call.

Reply:
that's because Automatic Renaming for RD's is not enabled in regression org; it is in qa_flow only. When automatic renaming is not on, that field will not show up on the modal form.

Reply:
spoke through slack and got more clarity.